FishBun/src/main/java/com/sangcomz/fishbun/permission/PermissionCheck.java

```diff
@@ -32,19 +34,21 @@ public boolean CheckStoragePermission() {
                 Manifest.permission.READ_EXTERNAL_STORAGE);
         int permissionCheckWrite = ContextCompat.checkSelfPermission(context,
                 Manifest.permission.WRITE_EXTERNAL_STORAGE);
-        if (permissionCheckRead != PackageManager.PERMISSION_GRANTED || permissionCheckWrite != PackageManager.PERMISSION_GRANTED) {
+        int cameraPermission = ContextCompat.checkSelfPermission(context,
```

Comment:
This method name needs to be modified. 
Until now, method were only requesting permission for storage, but now method are requesting permission for camera as well.

Reply:
Ok will change the method accordingly and push

Reply:
I have pushed the code with the method name changes